src/applications/simple-forms/21-0845/config/submit-transformer.js

```diff
@@ -0,0 +1,38 @@
+import { transformForSubmit as formsSystemTransformForSubmit } from 'platform/forms-system/src/js/helpers';
+
+import { LIMITED_INFORMATION_ITEMS } from '../definitions/constants';
+
+const escapedCharacterReplacer = (_key, value) => {
+  if (typeof value === 'string') {
+    return value
+      .replaceAll('"', "'")
+      .replace(/(?:\r\n|\n\n|\r|\n)/g, '; ')
+      .replace(/(?:\t|\f|\b)/g, '')
+      .replace(/\\(?!(f|n|r|t|[u,U][\d,a-fA-F]{4}))/gm, '/');
+  }
+
+  return value;
+};
+
+export default function transformForSubmit(formConfig, form) {
```

Comment:
whats the context of what needed to be changed here and why? just curious.

Reply:
Old checkbox-group from an older existing form saved limitedInformationItems as a comma-separated string of label-texts, so this transformer just converts the new data object of keys & booleans back into a comma-separated string.  This way Eric doesn't have to do extra work refactoring vets-api.